src/System.Management.Automation/engine/Modules/ModuleCmdletBase.cs

```diff
@@ -447,7 +447,16 @@ internal PSModuleInfo LoadUsingMultiVersionModuleBase(string moduleBase, Manifes
         /// <summary>
```

Comment:
@vexx32, test results were not published for PowerShell-CI-linux at vstfs:///Build/Build/25574